### DISCUSSION TITLE
Windows 11 - Unable to display "Windows 10 Start Menu"

### DISCUSSION THREAD

**Discussion by jeablu** [valinet/ExplorerPatcher#3692] [Q&A]
Hi,

I am on a PC with Windows 11. I wish to find the Windows 10 Start Menu with its dynamic tiles that I prefer.
On the internet I found procedures to find the Windows 10 Start menu:

**1**- Either by correcting the Windows registry (manually or by running the file "restore-start-menu-classic-windows-11.reg" (Malekal site)

**2**-Or with the "**ExplorerPatcher**" utility (ep_setup.exe), which is what I did and which worked.
The ExplorerPatcher utility installs a "Properties" file in which you can select "the Start menu style" by checking Windows 10.

**Problem**: after a time of operation of 2 weeks without problem. When starting the PC, the Start menu has become the original one of Windows 11 (I thing following a Windows update).
ExplorerPatcher is still installed but in the properties of the utility the option line "the Start menu style" has disappeared not allowing to choose Windows 10.

I checked the Windows registry, it is indeed modified to accept Windows 10. I have redone procedures 1 and 2 above in vain.

Have you encountered this problem, especially if you use the ExplorerPatcher utility: I do not understand why in ExplorerPatcher properties the line option "the style of the Win 10 or Win 11 start menu" has disappeared?

Thanks for your help.
ExplorerPatcher version 22621.3527.65.5

jeablu
PS- restoring Windows to an earlier date did not solve the problem.

**Comment by pyrates999** [comment]
Update ExplorerPatcher.

**Reply by jeablu** [reply]
RESOLVED - Unable to display "Windows 10 Start Menu
I install a last version of ExplorerPatcher  2261.3880.66.5 with defender disabled.
in properties, the option line "the Start menu style" displays and allows you to choose Windows 10.
It ok , "Start menu winsows 10" displays
jeablu
